dispatcher/MongoConnect.py

```diff
@@ -203,6 +198,7 @@ def aggregate_status(self):
                     'pll_unlocks': 0,
                     'number': -1}
                 for k in self.dc}
+        phys_stat = {}
```

Comment:
```suggestion
        phys_stat = defaultdict(list)  # list of statussus of each of the phys. detectors
```

Reply:
Or `phys_stat = {k:[] for k in self.dc}`